watchfrr/watchfrr.c

```diff
@@ -66,21 +66,7 @@
 /* Needs to be global, referenced somewhere inside libfrr. */
 struct thread_master *master;
 
-typedef enum {
```

Comment:
Do we need a period of deprecation?( I am not saying we do, just asking for opinions )